docs/site/Transactions.md

```diff
@@ -172,6 +172,24 @@ on transaction objects are as follows:
   rollback(): Promise<void>;
 ```
 
+### Checking Activeness
+
+For SQL connectors, function `isActive()` is exposed to return the activeness of
+the transaction by checking the existence of field `connection`. It is also
+available on the transaction objects:
+
+```ts
+  /**
+   * Check if the transaction has an active connection
+   */
+  isActive(): boolean;
+```
+
+Suppose you have a transaction called `tx`, you can call `tx.isActive()` to
+check whether it's still active.
+
+{% include note.html content=" The postgresql connector doesn't fully support this function due to its special way of creating the transaction instance. Details are explained in [issue #411](https://github.com/strongloop/loopback-connector-postgresql/issues/411). " %}
```

Comment:
Could you please elaborate on "The postgresql connector doesn't fully support this function"? 
And there's an extra space after the quote.

Reply:
@dhmlau The details are explained in [issue #411](https://github.com/strongloop/loopback-connector-postgresql/issues/411), or do you want me to expand the content here?

The function itself is simple, and I already have a brief explanation for the inconsistency:
> The postgresql connector doesn't fully support this function due to its special way of creating the transaction instance.

Reply:
I'm thinking whether we could add a one-liner about what doesn't really work in the postgresql connector, and then point to the issue for more details.

Reply:
@dhmlau added